ts/components/core/logos/LogoPayment.tsx

```diff
@@ -37,8 +38,30 @@ export const IOPaymentLogos = {
   applePay: LogoPaymentApplePay
 } as const;
 
+export const IOPaymentLogosUpperCased: { [key: string]: IOLogoPaymentType } = {
+  MAESTRO: "maestro",
+  MASTERCARD: "mastercard",
+  VISA: "visa",
+  AMEX: "amex",
+  DINERS: "diners",
+  DISCOVER: "discover",
+  JCB: "jcb",
+  UNIONPAY: "unionPay",
+  PAYPAL: "payPal",
+  BANCOMATPAY: "bancomatPay",
+  VPAY: "vPay",
+  PAGOBANCOMAT: "pagoBancomat",
+  SATISPAY: "satispay",
+  POSTEPAY: "postepay",
+  MYBANK: "myBank",
+  APPLEPAY: "applePay"
+} as const;
+
 export type IOLogoPaymentType = keyof typeof IOPaymentLogos;
 
+export const IOLogoPaymentTypeDecodable =
+  keysDecoderFromObject<IOLogoPaymentType>(IOPaymentLogos);
+
```

Comment:
It returns an object with all null values. Why?

Reply:
`io-ts`'s `is` and `decode` function are made for much more complex objects. If you want to check if a string is of the same type of a type literal you can use a simple type assertion, which comes with typescript!